helm/node-operator-chart/templates/rbac.yaml

```diff
@@ -26,6 +26,7 @@ rules:
       - core.giantswarm.io
     resources:
       - storageconfigs
```

Comment:
So this means the operator has also privileges to access the storageconfigs, which it does not. So we can remove this right? Considering this is all copied from the kvm-operator the kvm-operator should also get fixed because it does not access the storageconfigs AFAIK. Below is also access defined to kvmconfigs which is also wrong I guess. Looking at this file more closely shows that a lot of resources defined are not used at all and that also the nodes resource is missing, because the node-operator lists nodes. Thanks for the hints here. I will clean this up tomorrow.

Reply:
Oh, man. I thought you knew what did you put here :)
I just fixed error, which annoys my ops duty

Reply:
> Oh, man. I thought you knew what did you put here

I do not like your tone here Vol. Lets please not get down this road. Do not make me use your own techniques against you in the future. Lets please stay professional. I already acknowledged my mistake. No reason to kick in again. You could just care a little more and educate me to prevent such mistakes in the first place instead of doing finger pointing. As I said, I will fix this tomorrow. Thanks again for the important hint to understand the matter.